src/reflection.test.ts

```diff
@@ -48,26 +45,59 @@ describe("parseReflectionSchema", () => {
       EnumVal.addName(builder, nameOffset);
       // Make sure that we are testing an integer that will exceed the normal
       // precision bounds.
-      EnumVal.addValue(builder, BigInt(Number.MAX_SAFE_INTEGER) + BigInt(1));
+      EnumVal.addValue(builder, BigInt(Number.MAX_SAFE_INTEGER) + 2n);
```

Comment:
Was there any particular reason to change this to a 2 instead of a 1?

Reply:
Because
- `Number.MAX_SAFE_INTEGER` is 9007199254740991
- `Number.MAX_SAFE_INTEGER+1` is 9007199254740992
- `Number.MAX_SAFE_INTEGER+2` is also 9007199254740992